front/components/assistant_builder/InstructionScreen.tsx

```diff
@@ -329,17 +326,24 @@ function Suggestions({
   owner: WorkspaceType;
   instructions: string;
 }) {
-  const [suggestions, setSuggestions] = useState<BuilderSuggestionsType>(
-    !instructions
-      ? STATIC_SUGGESTIONS
-      : { status: "unavailable", reason: "irrelevant" }
+  // history of all suggestions. The first two are displayed.
+  const [suggestions, setSuggestions] = useState<string[]>(
+    !instructions ? STATIC_SUGGESTIONS : []
   );
+  const [suggestionsStatus, setSuggestionsStatus] = useState<
+    | "no_suggestions"
+    | "loading"
+    | "suggestions_available"
+    | "instructions_are_good"
+    | "error"
+  >(!instructions ? "suggestions_available" : "no_suggestions");
```

Comment:
Can we extract the suggestion status to a type? To keep the state easier to read/parse?